src/app/root/root.component.ts

```diff
@@ -108,4 +142,23 @@ export class RootComponent implements OnInit {
       mainContent.focus();
     }
   }
+
+  getBrowserName(): string {
+    const userAgent = this._window.nativeWindow.navigator.userAgent;
```

Comment:
@davide-negretti : After merging this PR, I'm now noticing that I can get an error on this line when run via SSR:
```
ERROR TypeError: Cannot read properties of undefined (reading 'userAgent')
    at RootComponent2.getBrowserName (C:\\dspace-angular\dist\server\main.js:1:1008203)
    at RootComponent2.ngOnInit (C:\\dspace-angular\dist\server\main.js:1:1006659)
    at callHookInternal (C:\\dspace-angular\dist\server\main.js:1:4134206)
```


To reproduce:
1. Build in prod mode : `yarn build:prod; yarn serve:ssr`
2. Access the homepage & that SSR error will appear in the console window where you ran `yarn serve:ssr`.

Could you possibly provide a quick fix so that I don't have to revert this PR?